Microsoft.Toolkit.Uwp.UI.Controls/MasterDetailsView/MasterDetailsView.cs

```diff
@@ -151,6 +141,15 @@ private void OnUnloaded(object sender, RoutedEventArgs e)
         }
 
         /// <summary>
+        /// If a selection is set, this updates the view to show it. By default, this is called anytime
+        /// a selection changes.
+        /// </summary>
+        protected virtual void ShowSelection()
```

Comment:
Good point. I can just check in the selection change handler if it's in master view only and then removes the need for this method. It did feel weird, but I couldn't figure out a way around it at the time. However just doing this now makes it so that mouse clicks don't work by default in the list. If I add an `IsItemClickEnabled` property then it can be opted in.